map_requests.py

```diff
@@ -40,15 +35,27 @@ def request_directions(start_location,end_location):
     start_location: tuple of floats (lat, lon) for starting point
     end_location: tuple of floats (lat, lon) for ending point
```

Comment:
Does it make sense for the inputs here to be Location objects instead of tuples?